pre_install_report/test/test_pre_install_report.py

```diff
@@ -5,7 +5,7 @@
 # ### Author: SAS Institute Inc.                                 ###
 ####################################################################
 #                                                                ###
-# Copyright (c) 2021, SAS Institute Inc., Cary, NC, USA.         ###
+# Copyright (c) 2022, SAS Institute Inc., Cary, NC, USA.         ###
```

Comment:
Probably should technically be 2021-2022 as there was a 2021 copyright prior.

Reply:
Modified code so that the output looks like this: "This release of Kubernetes is not supported 1.16.x"

Reply:
Corrected per chat with you. 2019-2022